src/components/ItemsList/ItemList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import ItemMenu from '../ItemMenu/ItemMenu'
+import {HeaderMenuItems} from "../values/HeaderMenuItems";
+
+class ItemsList extends React.Component {
+
+  render(){
+    let k = 0;
```

Comment:
you don't need parameter 'k' because map function take a callback (item, i) => ...